### PR TITLE
Remove unused transaction base props

### PR DESCRIPTION
**Pull request by Gudahtt**
These props were never given. They have been removed to simplify the component.

### CODE REVIEW

File: ui/app/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ export default class ConfirmPageContainerContent extends Component {
     assetImage: PropTypes.string,
```
Comment (danjm) [inline ui/app/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js:19]: Going to add review and comments file by file, to ensure my own thinking is good.

File: ui/app/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js [hunk 2]
```diff
@@ -88,23 +86,21 @@ export default class ConfirmPageContainerContent extends Component {
...
+          title={title}
+          titleComponent={titleComponent}
+          subtitle={subtitle}
+          subtitleComponent={subtitleComponent}
+          hideSubtitle={hideSubtitle}
+          identiconAddress={identiconAddress}
+          nonce={nonce}
+          assetImage={assetImage}
+        />
         {this.renderContent()}
```
Comment (danjm) [inline ui/app/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js:104]: This file looks good. `ConfirmPageContainerContent` is only used in `ConfirmPageContainer`, which is only used in `ConfirmTransactionBase`, the instances of which are never passed a summary component.

File: ui/app/components/app/confirm-page-container/confirm-page-container.component.js [hunk 3]
```diff
@@ -149,7 +147,6 @@ export default class ConfirmPageContainer extends Component {
             subtitle={subtitle}
             subtitleComponent={subtitleComponent}
             hideSubtitle={hideSubtitle}
-            summaryComponent={summaryComponent}
             detailsComponent={detailsComponent}
             dataComponent={dataComponent}
             errorMessage={errorMessage}
```
Comment (danjm) [inline ui/app/components/app/confirm-page-container/confirm-page-container.component.js:152]: This file looks good as per https://github.com/MetaMask/metamask-extension/pull/10295/files#r566206507

File: ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 4]
```diff
@@ -70,31 +70,19 @@ export default class ConfirmTransactionBase extends Component {
...
-    subtitle: PropTypes.string,
     subtitleComponent: PropTypes.node,
-    summaryComponent: PropTypes.node,
     title: PropTypes.string,
-    titleComponent: PropTypes.node,
-    valid: PropTypes.bool,
-    warning: PropTypes.string,
     advancedInlineGasShown: PropTypes.bool,
     insufficientBalance: PropTypes.bool,
     hideFiatConversion: PropTypes.bool,
```
Comment (danjm) [inline ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js:88]: changes to here in this file look good. I verified that none of these properties are passed to any of the instances of ConfrimTransactionBaseComponent

File: ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 5]
```diff
@@ -263,100 +244,90 @@ export default class ConfirmTransactionBase extends Component {
...
+                />
               </div>
             </div>
-          ) : null}
-        </div>
-      )
+          </div>
+        ) : null}
+      </div>
     )
```
Comment (danjm) [inline ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js:331]: above big group of changes is just indentation change

File: ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 6]
```diff
@@ -765,20 +708,18 @@ export default class ConfirmTransactionBase extends Component {
         action={functionType}
         title={title}
         titleComponent={this.renderTitleComponent()}
-        subtitle={subtitle}
```
Comment (danjm) [inline ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js:768]: I think subtitle can be removed from `ConfirmPageContainer` and down through the component tree, as was done with `summaryComponent`

File: ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 7]
```diff
@@ -789,7 +730,7 @@ export default class ConfirmTransactionBase extends Component {
         lastTx={lastTx}
         ofText={ofText}
         requestsWaitingText={requestsWaitingText}
-        disabled={!propsValid || !valid || submitting}
+        disabled={!valid || submitting}
         onEdit={() => this.handleEdit()}
         onCancelAll={() => this.handleCancelAll()}
         onCancel={() => this.handleCancel()}
```
Comment (danjm) [inline ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js:736]: this file looks good

**Review by danjm — CHANGES_REQUESTED** [review]
Left one suggested change: https://github.com/MetaMask/metamask-extension/pull/10295/files#r566221054

looks good otherwise

**Review by brad-decker — DISMISSED** [review]
LGTM

File: ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js (hunk 6 above)
Comment (Gudahtt) [reply ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js:768]: Good catch! Removed in 3b7176a8be50554766385e920dbe7dea3c5eb6d5

**Review by brad-decker — APPROVED** [review]
Still LGTM

**Review by danjm — APPROVED** [review]
LGTM!

**Pull request merged**